R/format_PEW_temp.R

```diff
@@ -0,0 +1,419 @@
+#' #' Construct standard summary for data from Peerdsbos West, Belgium.
+#' #' Actively started 27/11/2020
+#'
+#'
+#'
+#' #### ---------------------------------------------------------------------------------------~
+#' #### BEFORE CLOSING THE SCRIPT/PROJECT, COMMENT ALL ROWS, TO BE ABLE TO LOAD ALL FUNCTIONS NEXT TIME
+#' #### ---------------------------------------------------------------------------------------~
+#'
+#'
+#'
+#' # #### ---------------------------------------------------------------------------------------~
+#' # #### SHOULD BE BASED ON THE NEW V. 1.1.0 OF THE STANDARD PROTOCOL
+#' # #### GET ALL THE VARIABLES FROM THE PROTOCOL PDF
+#' # line <- readLines("C:/Users/ZuzanaZ/Dropbox/GITHUB_PROJECTS/SPI-Birds/notes_zz/prot_names_v110.txt")
+#' # prot_names <- gsub(pattern = "[[:digit:]]", replacement = "", x = line)
+#' # prot_names <- gsub(pattern = "\\.", replacement = "", x = prot_names)
+#' # prot_names <- trimws(prot_names)
+#' #
+#' # prot_names
+#' #
+#' # Individual_data_names <- prot_names[2:10]
+#' # Brood_data_names      <- prot_names[12:47]
+#' # Capture_data_names    <- prot_names[49:71]
+#' # Location_data_names   <- prot_names[73:81]
+#' # #### ---------------------------------------------------------------------------------------~
+#'
+#'
+#'
+#'
+#'
+#' #### ---------------------------------------------------------------------------------------~
+#' #### Questions/Doubts DATA OWNERS
+#' # What is "Neophobia Transponder"?
+#' # Code "Experiment"?
+#' # OriginalTarsusMethod
+#' # Check experimentID
+#'
+#' #### Questions/Doubts TEAM
+#' # What to do with strange ID, like "MetalRing&Tag_M2019NestK55" (34 individuals)?
+#' # What do do with individuals without ID?
+#' # Where does the chick-rearing data go? To Individual data?
+#'
+#' #### ---------------------------------------------------------------------------------------~
+#'
+#'
+#'
+#'
+#' library(pipelines)
```

Comment:
It might be easier to build this as a function from the start so you can add in the help documentation and record decisions you make as you work (e.g. exclude unknown individuals). That will also remove the need to comment out everything before you save.